src/types/spectodaClass.ts

```
@@ -0,0 +1,76 @@
+  ): Promise<any>;
+  disconnect(): Promise<void> | ReturnType<SpectodaRuntime["disconnect"]>;
+  connected(): Promise<null> | ReturnType<SpectodaRuntime["connected"]>;
+
+  // Network methods
+  assignOwnerSignature(ownerSignature: SpectodaTypes.NetworkSignature): boolean;
+  assignOwnerKey(ownerKey: SpectodaTypes.NetworkKey): boolean;
+  getOwnerSignature(): SpectodaTypes.NetworkSignature;
+  getOwnerKey(): SpectodaTypes.NetworkKey;
+
+  // Event handling
+  addEventListener<K extends keyof SpectodaJsEventMap>(event: K, callback: (props: SpectodaJsEventMap[K]) => void): Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)